import/import.js

```diff
@@ -1097,6 +1097,12 @@ function getCleanedRenderedHTML(html) {
     mutations++;
   });
 
+  $("div.in-page-callout").each((i, element) => {
+    $(element).removeClass("in-page-callout webdev");
```

Comment:
webdev??

Reply:
Does that actually work? What if an element only has `in-page-callout` or only `webdev`. Why isn't it:
```suggestion
    $(element).removeClass("in-page-callout");
    $(element).removeClass("webdev");
```
?

Reply:
The docs suggest that it would work? https://www.npmjs.com/package/cheerio#removeclass-classname-

`webdev` is innocent enough but, while we are at it 😄